app/code/Magento/InventoryConfigurableProductFrontend/view/frontend/web/js/configurable-variation-qty.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/**
+ * Configurable variation left qty.
+ */
+define([
+    'jquery',
+    'underscore',
+    'mage/url'
+], function ($, _, urlBuilder) {
+    'use strict';
+
+    return function (productId) {
+        var selectorInfoStockSkuQty = '.availability.only',
+            selectorInfoStockSkuQtyValue = '.availability.only > strong',
+            salesChannel = 'website',
+            productQtyInfoBlock = $(selectorInfoStockSkuQty),
+            productQtyInfo = $(selectorInfoStockSkuQtyValue);
+
+        if (!_.isUndefined(productId) && productId !== null) {
+            $.ajax({
+                url: urlBuilder.build('catalog/product/getQty/'),
```

Comment:
Why `catalog`? 

Ideally catalog entities should not be aware of the concept of Quantity, as this is a separate domain of inventory.